### DISCUSSION TITLE
Get a particular message

### DISCUSSION THREAD

**Discussion by safwanadnan** [devlikeapro/waha#562] [Q&A]
Is there a way to get a particular message using the message id?

**Comment by devlikepro** [comment]
not yeat https://github.com/devlikeapro/waha/issues/484
Comment with +1 to get notification when it's done!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
